linkerd/proxy/transport/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 //! Uses unsafe code to interact with socket options for keepalive and SO_ORIGINAL_DST.
```

Comment:
should maybe update this comment
```suggestion
//! Uses unsafe code to interact with socket options for SO_ORIGINAL_DST.
```